src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> newReport = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                newReport.add(line);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t to open this file", e);
+        }
+
+        List<String> result = statisticReport(newReport);
+        totalReport(result, toFileName);
```

Comment:
you've combined IO logic with business logic, which violates SRP principle. this method should be splitted into smallest ones like this:
```suggestion
        String dataFromFile = readFromFile(fromFileName);
        String report = processData(dataFromFile);
        writeToFile(report, toFileName);
```